examples/run_ransac.py

```diff
@@ -0,0 +1,85 @@
+"""
+=================
+Run RANSAC
+=================
+
+
+In this example we show how to run the RANSAC of ``pyprep``.
+"""
+
+# Authors: Yorguin Mantilla <yjmantilla@gmail.com>
+#
+# License: MIT
+# Based On: use_noisy_module.py
+
+###############################################################################
+# First we import what we need for this example.
+import numpy as np
+import mne
+from scipy import signal as sx
+import time as clk
+
+from pyprep.find_noisy_channels import NoisyChannels
+
+###############################################################################
+# Now let's make some arbitrary MNE raw object for demonstration purposes.
+# We will think of good channels as sine waves and bad channels correlated with
+# each other as sawtooths. The idea is that the RANSAC is able to identify
+# these channels. We will need to set a montage because the RANSAC needs to
+# interpolate.
```

Comment:
I suggest to rephrase using these two snippets:

We will simulate good channels as sine waves and will inject a few "bad" channels with sawtooth shape.

RANSAC will then identify the "bad" channels, because they will not be predicted well by a random subset of "good" channels.